src/test/java/org/jfrog/hudson/pipeline/integration/CommonITestsPipeline.java

```diff
@@ -353,11 +353,16 @@ void deletePropsTest(String buildName) throws Exception {
     void dockerPushTest(String buildName) throws Exception {
         try {
             // Get image name
-            String imageName = System.getenv("JENKINS_ARTIFACTORY_DOCKER_IMAGE");
-            if (StringUtils.isBlank(imageName)) {
-                throw new MissingArgumentException("Docker image name is not provided");
+            String domainName = System.getenv("JENKINS_ARTIFACTORY_DOCKER_DOMAIN");
+            if (StringUtils.isBlank(domainName)) {
+                throw new MissingArgumentException("Docker domain is not provided");
```

Comment:
Docker domain is not provided --> The JENKINS_ARTIFACTORY_DOCKER_DOMAIN environment variable is not set.

(this may make it easier to understand the issue and fix it).

Reply:
Done